v-next/hardhat-verify/src/internal/etherscan.ts

```diff
@@ -41,18 +44,26 @@ export class Etherscan {
   public apiUrl: string;
   public apiKey: string;
 
+  readonly #testDispatcher?: Dispatcher;
+  readonly #pollingIntervalMs: number;
+
   constructor(etherscanConfig: {
     chainId: number;
     name?: string;
     url: string;
-    apiUrl: string;
     apiKey: string;
+    testDispatcher?: Dispatcher;
```

Comment:
Can we rename this to `dispatcher` or `givenDispatcher` or something else.
I know our only intended usage is in tests, but I think constructor injection is valid approach, so we don't need to shamefully call it `testDispatcher`.